include/experimental/__p0009_bits/config.hpp

```diff
@@ -15,21 +15,25 @@
 //@HEADER
 #pragma once
 
-#ifndef __has_include
-#  define __has_include(x) 0
+#ifndef MDSPAN_IMPL_HAS_INCLUDE
```

Comment:
I switched this to our own macro rather than substituting in the standard one (since I think that could propagate a bug downstream if someone does `#ifdef __has_include`

Reply:
This is much better; thank you! : - )